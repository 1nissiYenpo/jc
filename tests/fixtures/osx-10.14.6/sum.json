[{"filename": "airport-I.json", "checksum": 38391, "blocks": 1}, {"filename": "airport-I.out", "checksum": 10927, "blocks": 1}, {"filename": "airport-s.json", "checksum": 19081, "blocks": 3}, {"filename": "airport-s.out", "checksum": 63504, "blocks": 2}, {"filename": "arp-a.json", "checksum": 60887, "blocks": 3}, {"filename": "arp-a.out", "checksum": 18964, "blocks": 1}, {"filename": "arp-a2.json", "checksum": 63795, "blocks": 3}, {"filename": "arp-a2.out", "checksum": 46674, "blocks": 2}, {"filename": "cksum.out", "checksum": 49539, "blocks": 5}, {"filename": "df-h.json", "checksum": 64766, "blocks": 2}, {"filename": "df-h.out", "checksum": 415, "blocks": 2}, {"filename": "df.json", "checksum": 37365, "blocks": 2}, {"filename": "df.out", "checksum": 12639, "blocks": 2}, {"filename": "dig-aaaa.json", "checksum": 8111, "blocks": 1}, {"filename": "dig-aaaa.out", "checksum": 50417, "blocks": 1}, {"filename": "dig-axfr.json", "checksum": 43183, "blocks": 6}, {"filename": "dig-axfr.out", "checksum": 28178, "blocks": 4}, {"filename": "dig-x.json", "checksum": 2727, "blocks": 1}, {"filename": "dig-x.out", "checksum": 11169, "blocks": 1}, {"filename": "dig.json", "checksum": 40945, "blocks": 2}, {"filename": "dig.out", "checksum": 61726, "blocks": 2}, {"filename": "du.json", "checksum": 38720, "blocks": 289}, {"filename": "du.out", "checksum": 16890, "blocks": 217}, {"filename": "file.json", "checksum": 37280, "blocks": 4}, {"filename": "file.out", "checksum": 9660, "blocks": 3}, {"filename": "file2.json", "checksum": 29539, "blocks": 5}, {"filename": "file2.out", "checksum": 33868, "blocks": 5}, {"filename": "group.json", "checksum": 50477, "blocks": 10}, {"filename": "group.out", "checksum": 55547, "blocks": 3}, {"filename": "id.json", "checksum": 21649, "blocks": 1}, {"filename": "id.out", "checksum": 26181, "blocks": 1}, {"filename": "ifconfig.json", "checksum": 17770, "blocks": 11}, {"filename": "ifconfig.out", "checksum": 10956, "blocks": 4}, {"filename": "ifconfig2.json", "checksum": 35645, "blocks": 12}, {"filename": "ifconfig2.out", "checksum": 38293, "blocks": 4}, {"filename": "last.json", "checksum": 36756, "blocks": 45}, {"filename": "last.out", "checksum": 23587, "blocks": 26}, {"filename": "ls-R-newlines.json", "checksum": 30013, "blocks": 1}, {"filename": "ls-R-newlines.out", "checksum": 42103, "blocks": 1}, {"filename": "ls-R.json", "checksum": 37108, "blocks": 459}, {"filename": "ls-R.out", "checksum": 63758, "blocks": 131}, {"filename": "ls-al.json", "checksum": 56923, "blocks": 5}, {"filename": "ls-al.out", "checksum": 15531, "blocks": 3}, {"filename": "ls-alR.json", "checksum": 1382, "blocks": 938}, {"filename": "ls-alR.out", "checksum": 4461, "blocks": 357}, {"filename": "ls-alh.json", "checksum": 57950, "blocks": 5}, {"filename": "ls-alh.out", "checksum": 6947, "blocks": 3}, {"filename": "ls-glob.json", "checksum": 31393, "blocks": 90}, {"filename": "ls-glob.out", "checksum": 1019, "blocks": 19}, {"filename": "ls-l-newlines.json", "checksum": 57505, "blocks": 2}, {"filename": "ls-l-newlines.out", "checksum": 41948, "blocks": 1}, {"filename": "ls-lR-empty-folder.json", "checksum": 11529, "blocks": 348}, {"filename": "ls-lR-empty-folder.out", "checksum": 58132, "blocks": 126}, {"filename": "ls-lR-newlines.json", "checksum": 6935, "blocks": 2}, {"filename": "ls-lR-newlines.out", "checksum": 41356, "blocks": 1}, {"filename": "ls-newlines.json", "checksum": 9894, "blocks": 1}, {"filename": "ls-newlines.out", "checksum": 57269, "blocks": 1}, {"filename": "ls.json", "checksum": 17338, "blocks": 1}, {"filename": "ls.out", "checksum": 52477, "blocks": 1}, {"filename": "md5.json", "checksum": 36019, "blocks": 8}, {"filename": "md5.out", "checksum": 42534, "blocks": 6}, {"filename": "mount.json", "checksum": 37000, "blocks": 1}, {"filename": "mount.out", "checksum": 16534, "blocks": 1}, {"filename": "mount2.json", "checksum": 45746, "blocks": 1}, {"filename": "mount2.out", "checksum": 55291, "blocks": 1}, {"filename": "netstat-Abn.json", "checksum": 34181, "blocks": 148}, {"filename": "netstat-Abn.out", "checksum": 5732, "blocks": 76}, {"filename": "netstat-An.json", "checksum": 27697, "blocks": 140}, {"filename": "netstat-An.out", "checksum": 26933, "blocks": 70}, {"filename": "netstat-i.json", "checksum": 13656, "blocks": 8}, {"filename": "netstat-i.out", "checksum": 7961, "blocks": 4}, {"filename": "netstat-r.json", "checksum": 11407, "blocks": 17}, {"filename": "netstat-r.out", "checksum": 18759, "blocks": 7}, {"filename": "netstat-rnl.json", "checksum": 63267, "blocks": 21}, {"filename": "netstat-rnl.out", "checksum": 62814, "blocks": 10}, {"filename": "netstat.json", "checksum": 41515, "blocks": 138}, {"filename": "netstat.out", "checksum": 32647, "blocks": 69}, {"filename": "passwd.json", "checksum": 39641, "blocks": 15}, {"filename": "passwd.out", "checksum": 19130, "blocks": 7}, {"filename": "ping-hostname-p.json", "checksum": 113, "blocks": 1}, {"filename": "ping-hostname-p.out", "checksum": 6247, "blocks": 1}, {"filename": "ping-hostname-s.json", "checksum": 12827, "blocks": 1}, {"filename": "ping-hostname-s.out", "checksum": 23515, "blocks": 1}, {"filename": "ping-hostname.json", "checksum": 21637, "blocks": 1}, {"filename": "ping-hostname.out", "checksum": 18604, "blocks": 1}, {"filename": "ping-ip-dup.json", "checksum": 26389, "blocks": 3}, {"filename": "ping-ip-dup.out", "checksum": 19769, "blocks": 2}, {"filename": "ping-ip-p.json", "checksum": 24589, "blocks": 1}, {"filename": "ping-ip-p.out", "checksum": 46644, "blocks": 1}, {"filename": "ping-ip-s.json", "checksum": 33523, "blocks": 1}, {"filename": "ping-ip-s.out", "checksum": 25608, "blocks": 1}, {"filename": "ping-ip.json", "checksum": 8854, "blocks": 1}, {"filename": "ping-ip.out", "checksum": 62691, "blocks": 1}, {"filename": "ping6-hostname-p.json", "checksum": 49568, "blocks": 1}, {"filename": "ping6-hostname-p.out", "checksum": 53253, "blocks": 1}, {"filename": "ping6-hostname-s.json", "checksum": 25633, "blocks": 1}, {"filename": "ping6-hostname-s.out", "checksum": 33091, "blocks": 1}, {"filename": "ping6-hostname.json", "checksum": 7678, "blocks": 1}, {"filename": "ping6-hostname.out", "checksum": 57086, "blocks": 1}, {"filename": "ping6-ip-dup.json", "checksum": 22863, "blocks": 10}, {"filename": "ping6-ip-dup.out", "checksum": 43494, "blocks": 6}, {"filename": "ping6-ip-p.json", "checksum": 39183, "blocks": 1}, {"filename": "ping6-ip-p.out", "checksum": 34802, "blocks": 1}, {"filename": "ping6-ip-s.json", "checksum": 40367, "blocks": 1}, {"filename": "ping6-ip-s.out", "checksum": 51554, "blocks": 1}, {"filename": "ping6-ip.json", "checksum": 57462, "blocks": 1}, {"filename": "ping6-ip.out", "checksum": 36941, "blocks": 1}, {"filename": "pip-list.json", "checksum": 29175, "blocks": 1}, {"filename": "pip-list.out", "checksum": 20474, "blocks": 1}, {"filename": "pip-show.json", "checksum": 37722, "blocks": 2}, {"filename": "pip-show.out", "checksum": 18260, "blocks": 1}, {"filename": "ps-axu.json", "checksum": 24887, "blocks": 97}, {"filename": "ps-axu.out", "checksum": 43405, "blocks": 59}, {"filename": "ps-ef.json", "checksum": 62822, "blocks": 70}, {"filename": "ps-ef.out", "checksum": 46842, "blocks": 48}, {"filename": "shasum.json", "checksum": 29690, "blocks": 12}, {"filename": "shasum.out", "checksum": 31405, "blocks": 8}, {"filename": "stat.json", "checksum": 30773, "blocks": 35}, {"filename": "stat.out", "checksum": 27996, "blocks": 16}, {"filename": "sysctl-a.json", "checksum": 54364, "blocks": 46}, {"filename": "sysctl-a.out", "checksum": 56073, "blocks": 42}, {"filename": "traceroute-asn.json", "checksum": 60232, "blocks": 2}, {"filename": "traceroute-asn.out", "checksum": 60268, "blocks": 1}, {"filename": "traceroute-mult-addresses.json", "checksum": 7554, "blocks": 1}, {"filename": "traceroute-mult-addresses.out", "checksum": 19479, "blocks": 1}, {"filename": "traceroute-no-header.json", "checksum": 14875, "blocks": 1}, {"filename": "traceroute-no-header.out", "checksum": 62949, "blocks": 1}, {"filename": "traceroute-q.json", "checksum": 12931, "blocks": 2}, {"filename": "traceroute-q.out", "checksum": 52433, "blocks": 1}, {"filename": "traceroute.json", "checksum": 45516, "blocks": 3}, {"filename": "traceroute.out", "checksum": 2209, "blocks": 1}, {"filename": "traceroute6-mult-addresses.json", "checksum": 12134, "blocks": 1}, {"filename": "traceroute6-mult-addresses.out", "checksum": 52070, "blocks": 1}, {"filename": "traceroute6.json", "checksum": 12134, "blocks": 1}, {"filename": "traceroute6.out", "checksum": 0, "blocks": 1}, {"filename": "uname-a.json", "checksum": 33535, "blocks": 1}, {"filename": "uname-a.out", "checksum": 39634, "blocks": 1}, {"filename": "uname.out", "checksum": 22645, "blocks": 1}, {"filename": "uptime.json", "checksum": 59092, "blocks": 1}, {"filename": "uptime.out", "checksum": 45958, "blocks": 1}, {"filename": "w.json", "checksum": 8647, "blocks": 1}, {"filename": "w.out", "checksum": 16304, "blocks": 1}, {"filename": "who-a.json", "checksum": 61006, "blocks": 1}, {"filename": "who-a.out", "checksum": 8784, "blocks": 1}, {"filename": "who.json", "checksum": 6467, "blocks": 1}, {"filename": "who.out", "checksum": 27498, "blocks": 1}]
