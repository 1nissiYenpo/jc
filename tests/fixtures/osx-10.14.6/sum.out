38391 1 airport-I.json
10927 1 airport-I.out
19081 3 airport-s.json
63504 2 airport-s.out
60887 3 arp-a.json
18964 1 arp-a.out
63795 3 arp-a2.json
46674 2 arp-a2.out
49539 5 cksum.out
64766 2 df-h.json
415 2 df-h.out
37365 2 df.json
12639 2 df.out
8111 1 dig-aaaa.json
50417 1 dig-aaaa.out
43183 6 dig-axfr.json
28178 4 dig-axfr.out
2727 1 dig-x.json
11169 1 dig-x.out
40945 2 dig.json
61726 2 dig.out
38720 289 du.json
16890 217 du.out
37280 4 file.json
9660 3 file.out
29539 5 file2.json
33868 5 file2.out
50477 10 group.json
55547 3 group.out
21649 1 id.json
26181 1 id.out
17770 11 ifconfig.json
10956 4 ifconfig.out
35645 12 ifconfig2.json
38293 4 ifconfig2.out
36756 45 last.json
23587 26 last.out
30013 1 ls-R-newlines.json
42103 1 ls-R-newlines.out
37108 459 ls-R.json
63758 131 ls-R.out
56923 5 ls-al.json
15531 3 ls-al.out
1382 938 ls-alR.json
4461 357 ls-alR.out
57950 5 ls-alh.json
6947 3 ls-alh.out
31393 90 ls-glob.json
1019 19 ls-glob.out
57505 2 ls-l-newlines.json
41948 1 ls-l-newlines.out
11529 348 ls-lR-empty-folder.json
58132 126 ls-lR-empty-folder.out
6935 2 ls-lR-newlines.json
41356 1 ls-lR-newlines.out
9894 1 ls-newlines.json
57269 1 ls-newlines.out
17338 1 ls.json
52477 1 ls.out
36019 8 md5.json
42534 6 md5.out
37000 1 mount.json
16534 1 mount.out
45746 1 mount2.json
55291 1 mount2.out
34181 148 netstat-Abn.json
5732 76 netstat-Abn.out
27697 140 netstat-An.json
26933 70 netstat-An.out
13656 8 netstat-i.json
7961 4 netstat-i.out
11407 17 netstat-r.json
18759 7 netstat-r.out
63267 21 netstat-rnl.json
62814 10 netstat-rnl.out
41515 138 netstat.json
32647 69 netstat.out
39641 15 passwd.json
19130 7 passwd.out
113 1 ping-hostname-p.json
6247 1 ping-hostname-p.out
12827 1 ping-hostname-s.json
23515 1 ping-hostname-s.out
21637 1 ping-hostname.json
18604 1 ping-hostname.out
26389 3 ping-ip-dup.json
19769 2 ping-ip-dup.out
24589 1 ping-ip-p.json
46644 1 ping-ip-p.out
33523 1 ping-ip-s.json
25608 1 ping-ip-s.out
8854 1 ping-ip.json
62691 1 ping-ip.out
49568 1 ping6-hostname-p.json
53253 1 ping6-hostname-p.out
25633 1 ping6-hostname-s.json
33091 1 ping6-hostname-s.out
7678 1 ping6-hostname.json
57086 1 ping6-hostname.out
22863 10 ping6-ip-dup.json
43494 6 ping6-ip-dup.out
39183 1 ping6-ip-p.json
34802 1 ping6-ip-p.out
40367 1 ping6-ip-s.json
51554 1 ping6-ip-s.out
57462 1 ping6-ip.json
36941 1 ping6-ip.out
29175 1 pip-list.json
20474 1 pip-list.out
37722 2 pip-show.json
18260 1 pip-show.out
24887 97 ps-axu.json
43405 59 ps-axu.out
62822 70 ps-ef.json
46842 48 ps-ef.out
29690 12 shasum.json
31405 8 shasum.out
30773 35 stat.json
27996 16 stat.out
54364 46 sysctl-a.json
56073 42 sysctl-a.out
60232 2 traceroute-asn.json
60268 1 traceroute-asn.out
7554 1 traceroute-mult-addresses.json
19479 1 traceroute-mult-addresses.out
14875 1 traceroute-no-header.json
62949 1 traceroute-no-header.out
12931 2 traceroute-q.json
52433 1 traceroute-q.out
45516 3 traceroute.json
2209 1 traceroute.out
12134 1 traceroute6-mult-addresses.json
52070 1 traceroute6-mult-addresses.out
12134 1 traceroute6.json
0 1 traceroute6.out
33535 1 uname-a.json
39634 1 uname-a.out
22645 1 uname.out
59092 1 uptime.json
45958 1 uptime.out
8647 1 w.json
16304 1 w.out
61006 1 who-a.json
8784 1 who-a.out
6467 1 who.json
27498 1 who.out
